services/movie.py

```diff
@@ -0,0 +1,36 @@
+from db.models import Movie
+from django.db.models.query import QuerySet
+from typing import List, Optional
+
+
+def get_movies(
+    genres_ids: Optional[List[int]] = None,
+    actors_ids: Optional[List[int]] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> QuerySet:
+    return Movie.objects.get(id=movie_id)
```

Comment:
Remember:`.all()`, `.filter()` returns QuerySet`.get()`, `.create()` returns model class, `-> User` for. example

Reply:
Fixed